static/app/components/tabs/tab.tsx

```diff
@@ -54,13 +54,24 @@ interface BaseTabProps {
    * by <DraggableTab> to pass in props used for drag-and-drop functionality.
    */
   additionalProps?: React.HTMLAttributes<HTMLElement>;
+  isTempTab?: boolean;
+  newVariant?: boolean;
```

Comment:
1. We probably want a better name here
2. Modifying the Tabs component to support he new variant is something that we can break out and make a separate PR to master for